src/main/java/it/gov/pagopa/merchant/service/MerchantServiceImpl.java

```diff
@@ -34,4 +37,8 @@ public MerchantListDTO getMerchantList(String organizationId,
                                            Pageable pageable) {
         return merchantListService.getMerchantList(organizationId, initiativeId, fiscalCode, pageable);
     }
+    @Override
+    public MerchantInfoDTO retrieveMerchantId(String fiscalCode, String acquirerId) {
+        return merchantIdService.getByFiscalCodeAndAcquirerId(fiscalCode, acquirerId);
```

Comment:
I'd use the same name for both the internal service's method and the facade method's one, just for the sake of clarity